_data-prepper/pipelines/configuration/sources/documentdb.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: default
+title: documentdb
+parent: Sources
+grand_parent: Pipelines
+nav_order: 2
+---
+
+# documentdb
+
+The `documentdb` source reads documents from [Amazon DocumentDB](https://aws.amazon.com/documentdb/) collections.
+It can read historical data from an export and keep up-to-date on the data using DocumentDB [Change Streams](https://docs.aws.amazon.com/documentdb/latest/developerguide/change_streams.html).
```

Comment:
Note: Amazon DocumentDB always requires "Amazon" to precede it, and AWS does not capitalize "change streams". Also, rather than "keep up to date", would "stay current with the data" work?